colcon_core/subprocess.py

```diff
@@ -80,30 +80,27 @@ async def run(
 
     rc, _, _ = await _async_check_call(
         args, stdout_callback, stderr_callback,
-        cwd=cwd, env=env, shell=shell, use_pty=use_pty)
+        use_pty=use_pty, **kwargs)
```

Comment:
Does it make sense to name the `kwargs` variable explicitly `other_popen_kwargs` as subprocess.run() does?

Reply:
I think it does, yes: a93eb41